deps/Versions.make

```diff
@@ -24,9 +24,7 @@ MBEDTLS_VER = 2.16.0
 MBEDTLS_BB_REL = 1
 LIBSSH2_VER = 1.9.0
 LIBSSH2_BB_REL = 1
-CURL_VER = 7.66.0
-CURL_BB_REL = 1
-LIBGIT2_VER = 0.28.5
+LIBGIT2_VER = 1.0.0
```

Comment:
Note to self: update to 1.0.1 in the next round of revisions (available since https://github.com/JuliaPackaging/Yggdrasil/pull/1410).
```suggestion
LIBGIT2_VER = 1.0.1
```